documentcloud/documents.py

```diff
@@ -11,10 +11,12 @@
 import re
 import warnings
 from functools import partial
+import mimetypes
 
 # Third Party
 from future.utils import python_2_unicode_compatible
 from requests.exceptions import RequestException
+import magic
```

Comment:
You need to add this as a dependency to use it.  But, I think it makes more sense to just go by file extension in the file name and not try to guess at a files actual type, so I would remove this.